packages/dd-trace/src/config.js

```diff
@@ -507,6 +507,9 @@ class Config {
     defaults['grpc.client.error.statuses'] = GRPC_CLIENT_ERROR_STATUSES
     defaults['grpc.server.error.statuses'] = GRPC_SERVER_ERROR_STATUSES
     defaults.headerTags = []
+    defaults['heapSnapshot.count'] = 0
+    defaults['heapSnapshot.folder'] = undefined
```

Comment:
Considering the two main scenarios (app is running on a staging/prod service, or app is running on a dev machine) I suppose the answer is not, but...

Should we provide a default directory? E.g. most (all?) systems (including Lambda) support writing to /tmp. I suppose that would be annoying for a macOS developer who needs to navigate to the directory using Finder. They'd probably prefer the snapshots be written to, say, the root of the project.

Reply:
The default is the root of the project which is why I left it undefined by default. I'm open to writing to other places, it just seemed the easiest to find by default.